go-server/go/api_default.go

```diff
@@ -0,0 +1,108 @@
+/*
+ * Web Server
+ *
+ * A simple web server for managing users
+ *
+ * API version: 1.0
+ * Contact: dmanor@redhat.com
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
+package swagger
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"path"
+)
+
+type userId int
+
+func AddUser(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+	w.WriteHeader(http.StatusOK)
+
+	err := r.ParseForm()
+
+	if err != nil {
+		log.Fatal(err)
+	}
+	name, email := r.Form.Get("name"), r.Form.Get("email")
+	db := connectDB("localhost", 5432, "dmanor", "D1234", "webserverdb")
```

Comment:
Few things regarding this line:

- It is better to keep the DB connection arguments elsewhere rather than duplicate them over and over on each function.
- Think what would happen if I'm trying to run this server on my laptop and port 5432 is already in use?
- Hard-coded credentials are good for local use and first POCs, but never push them into GitHub (especially that this repository is public)